prowler/providers/github/services/repository/repository_code_changes_multi_approval_requirement/repository_code_changes_multi_approval_requirement.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+from prowler.lib.check.models import Check, Check_Report_Github
+from prowler.providers.github.services.repository.repository_client import (
+    repository_client,
+)
+
+
+class repository_code_changes_multi_approval_requirement(Check):
+    """Check if a repository enforces at least 2 approvals for code changes
+
+    This class verifies whether each repository enforces at least 2 approvals for code changes.
+    """
+
+    def execute(self) -> List[Check_Report_Github]:
+        """Execute the Github Repository code changes enforce multi approval requirement check
+
+        Iterates over each repository and checks if the repository enforces at least 2 approvals for code changes.
+
+        Returns:
+            List[Check_Report_Github]: A list of reports for each repository
+        """
+        findings = []
+        for repo in repository_client.repositories.values():
+            report = Check_Report_Github(
+                metadata=self.metadata(), resource_metadata=repo
+            )
+            report.status = "FAIL"
+            report.status_extended = f"Repository {repo.name} does not enforce at least 2 approvals for code changes."
+
+            if repo.approval_count >= 2:
```

Comment:
The CIS 1.1.3 control asks for two reviews, but in small orgs this can be problematic.

Should we make this configurable, with 2 reviews as default?

Reply:
I thought about that, but didn't want to leave small teams without the approval check.

For me this check is more "compliance" than "security", because there's no way we can guarantee that who approved it did a proper review.

Either way, we can release it like this and revisit this when we have more feedback from the community.